roles/scaffold_rm_facts/templates/module_utils/network_os/facts/resource/resource.py.j2

```diff
@@ -58,6 +58,8 @@ class {{ resource|capitalize }}Facts(FactsBase):
                 obj = self.render_config(self.generated_spec, resource)
                 if obj:
                     objs.append(obj)
+
+        self.ansible_facts['ansible_network_resources'].pop('{{ resource }}', None)
```

Comment:
```suggestion
        ansible_facts['ansible_network_resources'].pop('{{ resource }}', None)
```

Reply:
@NilashishC ansible_facts is accessible with self only.